lib/firenest/pub_sub.ex

```diff
@@ -12,8 +12,8 @@ defmodule Firenest.PubSub do
   alongside the desired topology:
 
       children = [
-        supervisor(Firenest.Topology, [MyApp.Topology, adapter: Firenest.Topology.Erlang]),
-        supervisor(Firenest.PubSub, [MyApp.PubSub, topology: MyApp.Topology])
+        {Firenest.Topology, [MyApp.Topology, adapter: Firenest.Topology.Erlang]},
```

Comment:
We should change those to always receive a keyword list of options. Something like this:

    {Firenest.Topology, name: MyApp.Topology, adapter: Firenest.Topology.Erlang}

Where `name` and `adapter` are required fields. The same for `Firenest.PubSub`. You don't need to change the Firenest.Monitor though because that's going away. :)

Reply:
Then we also need to update the docs for those two!

Reply:
Just did. I think I got through to every bit of documentation as well, but it would be best to double check.